### PR TITLE
material icon migration for login component

### PR DESCRIPTION
**Pull request by avzz-19**

### CODE REVIEW

File: app/components/login-component/index.scss [hunk 1]
```diff
@@ -116,3 +116,14 @@
...
 }
+
+.ak-button-loader {
+  display: inline-flex;
+  align-items: center;
+  margin-right: 0.5em;
+  margin-left: 0.5em;
+
+  :global(.ak-loader-circle-indicator) {
+    stroke: var(--ak-button-color-primary-contrast-text) !important;
```
Comment (future-pirate-king) [inline app/components/login-component/index.scss:127]: use component specific variable name

File: app/styles/_icons.scss [hunk 2]
```diff
@@ -83,10 +83,6 @@
   @extend .mi-arrow-right;
 }
 
-.ak-icon-close {
```
Comment (future-pirate-king) [inline app/styles/_icons.scss:86]: why is this removed ?

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/styles/_icons.scss (hunk 2 above)
Comment (avzz-19) [reply app/styles/_icons.scss:86]: duplicate

File: app/styles/_component-variables.scss [hunk 3]
```diff
@@ -702,4 +702,7 @@ body {
   --dynamic-scan-modal-alert-background-color: var(--warn-light);
   --dynamic-scan-modal-alert-text-color: var(--warn-dark);
   --dynamic-scan-modal-alert-border-color: var(--warn-dark);
+
+  // variables for login-component
+  --ak-spinner-primary-contrast-text: var(--primary-contrast-text)
```
Comment (future-pirate-king) [inline app/styles/_component-variables.scss:707]: variable name should start with component name eg: `login-component-button-loader-color`

**Review by future-pirate-king — APPROVED** [review]

**Pull request merged**